src/components/CodeExample/CodeExample.js

```diff
@@ -61,7 +61,7 @@ class CodeExample extends Component {
         {loaded ? (
           <CodeEditor code={code} containerNodeID={containerNodeID} />
         ) : (
-          <h4>Loading code example...</h4>
+          <h4>Ներբեռնվում է կոդի օրինակը...</h4>
```

Comment:
Կարողա՞ "կոդի օրինակի ներբեռնում" ասենք, որ անգլի պես անորոշ հնչի, ոչ թե "Code example is loading"

Reply:
համաձայն